ibm/resource_ibm_iam_service_id.go

```diff
@@ -60,88 +55,130 @@ func resourceIBMIAMServiceID() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Set:      schema.HashString,
 			},
+			"unique_instance_crns": {
```

Comment:
Can you check a complete flow like create a service id and attach a service id policy to any service instance ...in the next run can you check will those values will be stored here (shd we make it as computed also)